.github/workflows/create-release-build.yaml

```diff
@@ -0,0 +1,72 @@
+# This starter workflow is for a CMake project running on a single platform.
+name: Create Release Build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+  # Manual trigger using the Actions page.
+  workflow_dispatch:
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build_release:
+    runs-on: ubuntu-22.04-8core
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
+        ref: '${{github.ref}}'
+
+    - name: Install Dependencies
+      run: sudo apt-get -y install libunwind-dev libgflags-dev libssl-dev libelf-dev protobuf-compiler libzstd-dev
+
+    - name: Configure CMake
+      # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+      # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+      run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DENABLE_TOOL=LLVM -S ${{github.workspace}}
+
+    - name: Build
+      # Build your program with the given configuration
+      run: make -C ${{github.workspace}}/build -j6
```

Comment:
Given that we have 8 cores, maybe -j8 or even -j12 is ok?

Reply:
Replaced with -j8